sdk/cdn/Azure.ResourceManager.Cdn/src/autorest.md

```diff
@@ -89,6 +89,9 @@ directive:
                         if(key.endsWith('ActionParameters')) {
                              $[key]['x-ms-client-name'] = newKey + 'Properties';
                         }
+                        if(key.endsWith('ConditionParameters')) {
+                            $[key].properties.operator['x-ms-client-name'] = 'op';
```

Comment:
Is this change from comments of arch board? this name reads weird

Reply:
Yes, the main reason of this change is that arch board want to avoid the occurrence of `@operator` in the method parameters, here is the [comment](https://apiview.dev/Assemblies/Conversation/43b0983f23fe4e07b9acdd88d99492c6#517929f6ed4d4b1c80f9f5c50c8c0ad7). Unfortunately I didn't find a way to only change the parameter name in model ctor, and I also don't want to change generator to make exemption for this edge case, so I just renamed it.